CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Breakpoint support through `XCBreakpointList`. https://github.com/xcodeswift/xcproj/pull/172 by [@alexruperez](https://github.com/alexruperez)
 - Add convenience method to find targets with a given name https://github.com/xcodeswift/xcproj/pull/184 by @pepibumur.
 - Danger plugin that fails earlier if files have been added/deleted and the Carthage project hasn't been regenerated afterwards https://github.com/xcodeswift/xcproj/pull/187 by @pepibumur.
+- Add breakpoint `condition` parameter by [@alexruperez](https://github.com/alexruperez).
```

Comment:
This is in the wrong version. The 1.7.0 release heading was never added, but I've just fixed it in master. You might have to rebase https://github.com/xcodeswift/xcproj/commit/b081b1c6858e028c75a8d6fae41b01b206252a48

Reply:
Rebased, thanks! :)